src/direct-linking/content_script/annotations/anchoring/html.coffee

```diff
@@ -81,7 +76,11 @@ exports.anchor = (root, selectors, options = {}) ->
       return querySelector(TextPositionAnchor, root, position, options)
       .then(maybeAssertQuote)
 
-  
+  if quote?
```

Comment:
What's the change in this module for? (not necessarily a problem, just want to understand it more in-case it's unintended conflict res mistake or something)

Reply:
Not sure about this, I think it comes from conflict resolution.

Reply:
I would then recommend reverting this change when you remove the `src/bookmarks/background` modules.

Reply:
Done! :+1: